events/handlers.js

```diff
@@ -0,0 +1,184 @@
+const path = require("path");
+const createCsvWriter = require("csv-writer").createObjectCsvWriter;
+const csv = require("csv-parser");
+const fs = require("fs");
+const converter = require("json-2-csv");
+const { EOF } = require("dns");
+const lockFile = require("lockfile");
+
+const filePath = path.join(__dirname, "events.csv");
+const headerFile = [
+  { id: "id", title: "id" },
+  { id: "title", title: "title" },
+  { id: "location", title: "location" },
+  { id: "timestamp", title: "timestamp" },
+];
+
+function getEventById(req, res) {
+  lockEvents(req, res, (req, res) => {
+    let event = {};
+    fs.createReadStream(filePath)
+      .pipe(csv())
+      .on("data", (data) => {
+        // if we found one row no need to check others
+        if (Object.keys(event).length !== 0) {
+          return;
+        }
+        if (data.id == req.params.eventId) {
+          event = data;
+        }
+      })
+      .on("end", () => {
+        if (Object.keys(event).length === 0) {
+          res.status(404).send("event not found");
+        } else {
+          res.send(event);
+        }
+      });
+  });
+}
+
+function getEvents(req, res) {
```

Comment:
It would be nice if we could move methods that work with csv data(files) to another file/module. We would divide responsibilities between modules/files. One file would be responsible for the HTTP layer(handler) and another one for data/csv layer.